pbs/pbsrequest.go

```diff
@@ -30,20 +37,42 @@ type Bids struct {
 	Params     json.RawMessage `json:"params"`
 }
 
+type PBSVideo struct {
+	Mimes          []string `json:"mimes,omitempty"`
+	Minduration    uint64   `json:"minduration,omitempty"`
+	Maxduration    uint64   `json:"maxduration,omitempty"`
+	Startdelay     int64    `json:"startdelay,omitempty"`
+	Skippable      int      `json:"skippable,omitempty"`
+	PlaybackMethod int8     `json:"playback_method,omitempty"`
+	Frameworks     []string `json:"frameworks,omitempty"`
+}
+
 type AdUnit struct {
-	Code     string           `json:"code"`
-	TopFrame int8             `json:"is_top_frame"`
-	Sizes    []openrtb.Format `json:"sizes"`
-	Bids     []Bids           `json:"bids"`
-	ConfigID string           `json:"config_id"`
+	Code       string           `json:"code"`
+	TopFrame   int8             `json:"is_top_frame"`
+	Sizes      []openrtb.Format `json:"sizes"`
+	Bids       []Bids           `json:"bids"`
+	ConfigID   string           `json:"config_id"`
+	MediaTypes []string         `json:"media_types"`
 }
 
 type PBSAdUnit struct {
-	Sizes    []openrtb.Format
-	TopFrame int8
-	Code     string
-	BidID    string
-	Params   json.RawMessage
+	Sizes      []openrtb.Format
+	TopFrame   int8
+	Code       string
+	BidID      string
+	Params     json.RawMessage
+	Video      PBSVideo
+	MediaTypes []MediaType
```

Comment:
I think we have to go back to the spec on this one. According to ORTB 2.5, to represent a video impression, the `video` object is sent. If we examine the video object we see that it requires at minimum a `mimes`  array type. That would seem to indicate to me that for our implementation a request with `MediaTypes` containing `video` without a corresponding `Video` object would be invalid. 
![image](https://user-images.githubusercontent.com/1870166/29461587-420d621e-83fa-11e7-9a22-bbf529d35600.png)

http://www.iab.com/wp-content/uploads/2016/03/OpenRTB-API-Specification-Version-2-5-FINAL.pdf